### PR TITLE
added priority buttons

### PR DESCRIPTION
**Pull request by erikgus**
added sorting on priority
added filter by subtopic

### CODE REVIEW

**Review by johnnadeluy — CHANGES_REQUESTED** [review]
I also noticed that the title of the part is hardcoded. Maybe that should be added to phrases?

File: src/main/resources/site/content-types/article/article.xml [hunk 1]
```diff
@@ -238,10 +238,14 @@
 					</label>
 					<occurrences minimum="0" maximum="1" />
 				</input>
-				<input name="showOnFrontPage" type="CheckBox">
-					<label>
-						Vis på forside
-					</label>
+				<input name="frontPagePriority" type="RadioButton">
```
[the remark this answers is not in the document]
Comment (erikgus) [reply src/main/resources/site/content-types/article/article.xml:241]: Sort dont work on false and true, but will work on 0 and 1

File: src/main/resources/site/parts/articleList/articleList.ts [hunk 2]
```diff
@@ -55,21 +55,13 @@ function getArticles(language: string): QueryResponse<Article> {
...
-              field: 'data.showOnFrontPage'
-            }
-          },
-          {
-            hasValue: {
-              field: 'data.showOnFrontPage',
-              values: [
-                true
-              ]
+              field: 'data.subtopic'
```
[the remark this answers is not in the document]
Comment (erikgus) [reply src/main/resources/site/parts/articleList/articleList.ts:64]: isnt that a field that all content have?

[the remark this answers is not in the document]
Comment (johnnadeluy) [reply src/main/resources/site/parts/articleList/articleList.ts:68]: Not necessarily I think 🤔 Wasn't there an issue with newly made articles in draft?

**Review by johnnadeluy — APPROVED** [review]

**Pull request merged**